### ISSUE TITLE
Dropping temp views doesn't work without fully qualifying

### ISSUE THREAD

**Issue by JelteF** [citusdata/citus#6002]
```sql
CREATE TEMP VIEW v AS select 1;
-- WARNING:  0A000: "view v" has dependency on unsupported object "schema pg_temp_8"
-- DETAIL:  "view v" will be created only locally
-- LOCATION:  DeferErrorIfHasUnsupportedDependency, dependency.c:899
-- CREATE VIEW
DROP VIEW v;
-- ERROR:  42P01: view "v" does not exist
-- LOCATION:  DropErrorMsgNonExistent, tablecmds.c:1261
DROP VIEW pg_temp_8.v;
-- SUCCESS;
```

The unqualified drop works in plain postgres

**Comment by JelteF** [comment]
This is a regression in Citus 11.0, it is not present in Citus 10.2

**Comment by onderkalaci** [comment]
fixed by #6003
